core-test/src/test/scala/org/bitcoins/core/crypto/NativeSecp256k1ScalaTest.scala

```diff
@@ -0,0 +1,73 @@
+package org.bitcoins.core.crypto
+
+import org.bitcoin.NativeSecp256k1
+import org.bitcoins.core.util.CryptoUtil
+import org.bitcoins.testkit.core.gen.NumberGenerator
+import org.bitcoins.testkit.util.BitcoinSUnitTest
+import scodec.bits.ByteVector
+
+class NativeSecp256k1ScalaTest extends BitcoinSUnitTest {
+  behavior of "NativeSecp256k1"
+
```

Comment:
Since this is new code I would 

```scala
implicit override val generatorDrivenConfig = generatorDrivenConfigNewCode
```

That makes it so that we execute the property based tests 100 times instead of 10.